Source/NimbleCommander/NimbleCommander/Resources/Config.json

```diff
@@ -866,7 +866,205 @@
+                        "filePanelsTabsSelectedKeyWndActiveBackgroundColor": "#FFFFFF",
+                        "filePanelsTabsSelectedKeyWndInactiveBackgroundColor": "#F8F8F8",
+                        "filePanelsTabsSelectedNotKeyWndBackgroundColor": "#FFFFFF",
+                        "filePanelsTabsRegularKeyWndHoverBackgroundColor": "#E3E3E3",
+                        "filePanelsTabsRegularKeyWndRegularBackgroundColor": "#F0F0F0",
+                        "filePanelsTabsRegularNotKeyWndBackgroundColor": "#FFFFFF00",
+                        "filePanelsTabsSeparatorColor": "#DEDEDE",
+                        "filePanelsTabsPictogramColor": "#586069",
+                        "filePanelsHeaderFont": "@labelFont, 11",
+                        "filePanelsHeaderTextColor": "#586069",
+                        "filePanelsHeaderActiveTextColor": "#586069",
+                        "filePanelsHeaderActiveBackgroundColor": "#FFFFFF",
+                        "filePanelsHeaderInactiveBackgroundColor": "#FFFFFF00",
+                        "filePanelsHeaderSeparatorColor": "#DEDEDE",
+                        "filePanelsFooterFont": "@labelFont, 11",
+                        "filePanelsFooterTextColor": "#586069",
+                        "filePanelsFooterActiveTextColor": "#586069",
+                        "filePanelsFooterSeparatorsColor": "#DEDEDE",
+                        "filePanelsFooterActiveBackgroundColor": "#FFFFFF",
+                        "filePanelsFooterInactiveBackgroundColor": "#E8E8E8",
+                        "filePanelsListFont": "@systemFont, 13",
+                        "filePanelsListGridColor": "#FFFFFF00",
+                        "filePanelsListHeaderFont": "@systemFont, 11",
+                        "filePanelsListHeaderBackgroundColor": "#FFFFFF",
+                        "filePanelsListHeaderTextColor": "#586069",
+                        "filePanelsListHeaderSeparatorColor": "#DEDEDE",
+                        "filePanelsListFocusedActiveRowBackgroundColor": "#0064E1",
+                        "filePanelsListFocusedInactiveRowBackgroundColor": "#0064E155",
+                        "filePanelsListSelectedItemBackgroundColor": "#FFF2BE",
+                        "filePanelsListRegularEvenRowBackgroundColor": "#FFFFFF",
+                        "filePanelsListRegularOddRowBackgroundColor": "#F6F6F6",
+                        "filePanelsBriefFont": "@systemFont, 13",
+                        "filePanelsBriefGridColor": "#E6E6E6",
+                        "filePanelsBriefRegularEvenRowBackgroundColor": "#FFFFFF",
+                        "filePanelsBriefRegularOddRowBackgroundColor": "#F6F6F6",
+                        "filePanelsBriefFocusedActiveItemBackgroundColor": "#0064E1",
+                        "filePanelsBriefFocusedInactiveItemBackgroundColor": "#0064E155",
+                        "filePanelsBriefSelectedItemBackgroundColor": "#FFF2BE",
+                        "terminalFont": "IBM Plex Mono, 12",
```

Comment:
That doesn't seem like something macOS ships by default?

Reply:
Ooh, true, missed that. I'll revert to Menlo.

Reply:
IIRC there are other monospaced fonts in macOS to try.

Reply:
fixed: https://github.com/mikekazakov/nimble-commander/pull/122/commits/58bf3bf8b2903db659b14669f968d6291d97f839